Packs/CortexXDR/ReleaseNotes/6_0_5.md

```diff
@@ -0,0 +1,26 @@
+
+#### Layouts
+
+##### Cortex XDR Incident
+
+- Added a field for Command Line Verdict.
```

Comment:
I dont see that field :)

Reply:
If omri added this field. Why you have it in your release notes?

Reply:
I have it in the release notes under `Layouts`, Omri added the field and used it in the `XDR Lite Incident` - Layout only and now I use it in the normal `XDR Incident` - Layout :)